src/TulsiGenerator/PBXTargetGenerator.swift

```diff
@@ -751,8 +754,9 @@ final class PBXTargetGenerator: PBXTargetGeneratorProtocol {
     // Bazel generates a symlink to a directory that collects all of the data needed for this
     // workspace. While this is often identical to the workspace, it sometimes collects other paths
     // and is the better option for most Xcode project path references.
+    // This directory is symlinked to `tulsi-workspace` during builds.
     buildSettings["\(PBXTargetGenerator.BazelWorkspaceSymlinkVarName)"] =
-        "${\(PBXTargetGenerator.WorkspaceRootVarName)}/\(bazelWorkspaceSymlinkPath)"
+        "${\(PBXTargetGenerator.WorkspaceRootVarName)}/\(PBXTargetGenerator.TulsiWorkspacePath)"
```

Comment:
Looks like this is the only use of `bazelWorkspaceSymlinkPath`, so you can get rid of the property.